submit_pyspark_job_to_driver_node_group_cluster.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
```

Comment:
This file needs to be moved to the `dataproc/snippets` folder.

Reply:
Also, please clarify that the existing "submit_spark_job_to_driver_node_group_cluster.py" sample is different to this "submit_**py**spark_job_to_driver_node_group_cluster.py` sample.